contributing.md

```diff
@@ -5,7 +5,7 @@ agree to abide by the Wagtail [code of conduct](https://github.com/wagtail/wagta
 
 If you do not have write access to the repo, fork, then clone the repo:
 
-    git clone git@github.com:your-username/bakerydemo.git
+    `$ git clone git@github.com:your-username/bakerydemo.git`
```

Comment:
This change doesn't seem to be correct - this is already a code block, so the backtick characters appear in the output. Also, we should leave out the `$` - it should already be clear that this is a command to enter at the command prompt, and inexperienced users might think that the `$` is part of the command.

Reply:
Thanks for the feedback! I've made some changes!